authentication/models.py

```diff
@@ -1,5 +1,106 @@
-""" Models for the authentication app."""
+""" This module contains models for the authentication app. """
 
 from django.db import models
 from django.contrib.auth.models import User
-# Create your models here.
+
+
+class Profile(models.Model):
+    """
+    Model representing a user profile.
+
+    Fields:
+       - user: The related user.
+       - skills: The list of skills associated with the profile.
+       - bio: The biography of the user.
+       - profile_picture : The profile picture of the user.
+       - github: The GitHub profile link of the user.
+       - linkedin: The LinkedIn profile link of the user.
+       - youtube: The YouTube profile link of the user.
+       - gender: The gender of the user.
+       - date_of_birth: The date of birth of the user.
+    """
+
+    user = models.OneToOneField(
+        to=User,
+        on_delete=models.CASCADE,
+        blank=True,
+        null=True,
+        help_text='The related user.'
+    )
+    skills = models.ManyToManyField(
+        'Skill',
+        blank=True,
+        null=True,
+        related_name='profile',
+        help_text='The skills associated with the profile.'
+    )
+    bio = models.TextField(
+        blank=True,
+        null=True,
+        help_text='A detailed bio of the user about himself.'
+    )
+    profile_picture = models.ImageField(
+        blank=True,
+        null=True,
+        help_text='A profile picture of the user.'
+    )
+    github = models.URLField(
```

Comment:
Github username can also be used.

Reply:
I was thinking to use these fields in the profile cards as href that's why I took them as URLField.